aws/resource_aws_dms_endpoint.go

```diff
@@ -279,14 +279,14 @@ func resourceAwsDmsEndpointCreate(d *schema.ResourceData, meta interface{}) erro
 		if v, ok := d.GetOk("database_name"); ok {
 			request.DatabaseName = aws.String(v.(string))
 		}
-		if v, ok := d.GetOk("extra_connection_attributes"); ok {
-			request.ExtraConnectionAttributes = aws.String(v.(string))
-		}
 		if v, ok := d.GetOk("kms_key_arn"); ok {
```

Comment:
While it's not related to `extra_connection_attributes` I also think the `kms_key_arn` is in the wrong place here. 

Its handled in the [state](https://github.com/terraform-providers/terraform-provider-aws/pull/5607/files#diff-af93466f07f9ddb37474bbef0d0cd873R571) outside of the default block and the [docs describe](https://docs.aws.amazon.com/dms/latest/userguide/CHAP_Security.EncryptionKey.html) this as the encryption key used by the replication instance and connection information so it shouldn't be service specific.

Reply:
Thanks for pointing that out. I'll take a closer look at it soon.